tests/integration/test_model.py

```diff
@@ -924,3 +924,77 @@ def test_experiment_automl_regressor(c, client, training_df):
     ), "Best model was not registered"
 
     check_trained_model(c, "my_automl_exp2")
+
+
+def test_predict_with_nullable_types(c):
+    df = pd.DataFrame(
+        {
+            "rough_day_of_year": [0, 1, 2, 3],
+            "prev_day_inches_rained": [0, 1, 2, 3],
+            "rained": [False, False, False, True],
+        }
+    )
+    c.create_table("train_set", df)
+
+    model_class = "'sklearn.linear_model.LogisticRegression'"
```

Comment:
Should add a `@pytest.mark.parametrize("gpu", [False, pytest.param(True, marks=pytest.mark.gpu)])` and test with `sklearn` and `cuml`.